internal/virtualmtime/virtualmtime.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2014 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package virtualmtime
+
+import (
+	"time"
+)
+
+type VirtualMtimeRepo interface {
```

Comment:
I don't know that this interface is technically required anymore, but I like the abstraction it provides so I left it in.